tests/test_datetime.py

```diff
@@ -62,7 +62,7 @@ async def test_duration_01(self):
         ''', durs)
         self.assertEqual(list(durs_from_db), durs)
 
-    @unittest.expectedFailure  # no relative delta in edgedb yet
+    @unittest.expectedFailure
```

Comment:
There is a bit of a chicken and egg problem here. The server updates need to be published before these tests should be expected to pass, but the server needs the changes in this PR for it's own test suite.